src/client/project/template_builder/helpers.py

```diff
@@ -287,9 +287,9 @@ class TranslateHelperV1(BaseHelperV1, spec_value=ComponentType.HELPER_TRANSLATE)
 
 
 class YandexDiskProxyHelperV1(BaseHelperV1, spec_value=ComponentType.HELPER_YANDEX_DISK_PROXY):
-    """You can use this component to download files from Yandex.Disk.
+    """You can use this component to download files from Yandex&#160;Disk.
```

Comment:
some troubles with encoding?

Reply:
@alexdrydew

Reply:
Not with encoding, but with yfm -> html conversion